sim/warlock/talents.go

```diff
@@ -236,19 +212,19 @@ func (warlock *Warlock) setupPyroclasm() {
 }
 
 func (warlock *Warlock) setupEradication() {
-	hasteBonusPercent := float64(warlock.Talents.Eradication) * 6
+	castSpeedPercentBonus := float64(warlock.Talents.Eradication) * 6
```

Comment:
This would be easier to read if you use `0.06` here and `0.02` for eradication, and then remove the `/100` in the aura.